remo/profiles/views.py

```diff
@@ -195,10 +195,10 @@ def view_profile(request, display_name):
             messages.info(request, mark_safe(msg))
 
     if nominee_form.is_valid():
-        if ((is_nomination_period or
-             waffle.switch_is_active('enable_rotm_tasks')) and
-                request.user.groups.filter(name='Mentor').exists()):
-            nominee_form.save()
+        if ((is_nomination_period or waffle.switch_is_active('enable_rotm_tasks')) and
+                request.user.groups.filter(name='Mentor').exists() and
+                request.user != user):
+            nominee_form.save(nominated_by=request.user)
```

Comment:
This method is not possible. The [RotmNomineeForm](https://github.com/mozilla/remo/pull/1272/files#diff-56f90107a7edcc544f3baacfcfca8906R274) is not a modelform and does not return any instance. If we try to return the instance, then ``commit=False`` can not be applied. Therefore if we try to do everything from view, we need to call two mysql query. So it will not be a good choice.